.circleci/config.yml

```diff
@@ -40,8 +46,8 @@ jobs:
             mkdir -p ~/.ssh
             ssh-keyscan -H github.com >> ~/.ssh/known_hosts
       - run: 
-          name: linking conda
-          command: ln -s /opt/conda $HOME/miniconda
+          name: Setup base system
+          command: echo '. /opt/docker/bin/entrypoint_source' >> $BASH_ENV
```

Comment:
Isn't this entrypoint the one with the gigantic chown? If so, why do we include it now (remember the performance problems)? Didn't we have a fix in the entrypoint of our derived image?

Reply:
The `chown` one is `/opt/docker/bin/entrypoint` which sources `/opt/docker/bin/entrypoint_source` and does all the user creation and `chown` stuff.
`/opt/docker/bin/entrypoint_source` is lightweight, but I'm actually not happy with hardcoding its path here. It would be better to turn the `docker-entrypoint` from the `bioconda-utils` repo into `docker-entrypoint` and `docker-entrypoint-source` and then use the latter here.
Well, even better would be to use `--docker`, but I'm not finished with some workaround to enable that.

Reply:
Nice to hear that --docker might eventually work!

Ok, sounds reasonable, but why not loading entrypoint_source directly in the bioconda-utils docker image?

Reply:
It is loaded in the docker image. But it only makes non-permanent changes `PATH`, i.e., doesn't store it in `BASH_ENV` & co. And I'm pretty sure `steps` in `.circleci/config.yml` will run `/bin/bash` instead of that entrypoint, though.

Reply:
I thought the whole point of entrypoints was that they run in addition to bin/bash. But I guess you checked that and discovered that the entrypoint is not loaded?

Reply:
> But I guess you checked that and discovered that the entrypoint is not loaded?

Correct, learned that by observation, not by deduction. And I'm still very confused about that. You can even override the Docker entrypoint with a custom one in the CircleCI config. But I still don't know when/how/if that is used (tested with `circleci build` locally).
It's also the reason why people couldn't run `git` until you added it to `bioconda-utils`'s requirements -- if the entry point would have been used, `git` from `/opt/conda/bin` would've been on `PATH`. 😕